### PR TITLE
WFNEWS-2411 Add highlights widget to dashboard

### PR DESCRIPTION
**Pull request by ssylver93**
Adding prevention-targeted blog entries to new dashboard Highlights widget.
Desktop;
image
Mobile;
image

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/highlights-widget/highlights-widget.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<div class="container">
+    <div class="widget-header">
+        <span class="widget-title">Highlights</span>
+    </div>
+    <div class="content">
+        <div class="highlight-card" *ngFor="let post of posts" (click)="openLink(post.link)">
```
Comment (sukhpalp) [inline client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/highlights-widget/highlights-widget.component.html]: Please make highlight-card a component

**Review by sukhpalp — CHANGES_REQUESTED** [review]
Almost there, but lets move the highlight-card into a self contained component.

File: client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/highlights-widget/highlights-widget.component.html (hunk 1 above)
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/highlights-widget/highlights-widget.component.html]: Done now. New screenshots after moving highlights-card to standalone component:
Desktop:
image
Mobile:
image

**Review by sukhpalp — APPROVED** [review]
Looks good!

**Pull request merged**